### ISSUE TITLE
WinPython-64bit-3.5.4.0Qt5 Spyder unresponsive at startup

### ISSUE THREAD

**Issue by jcbalmer** [spyder-ide/spyder#5119]
## Description

**What steps will reproduce the problem?**

1. Opening Spyder.exe
2. Will open to the main screen but will then be unresponsive immediately
3. Open windows task manager and under the Processes tab, find multiple pythonw.exe listed, delete a few of these until Spyder suddenly responds.
4. Sypder then responds but lags significantly.
5. After lagging for 30sec or so the following message displays in a Spyder Error popup (also see attached):

![image](https://user-images.githubusercontent.com/20058333/30007451-d19a0250-9162-11e7-9e03-6696e00d3404.png)

"Sorry for the inconvenience.
You can automatically sumit this error to our Github issues tracker
Note: You need a Github account for that."
  File "C:\Program Files\WinPython-64bit-3.5.4.0Qt5\python-3.5.4.amd64\lib\site-packages\spyder_terminal\terminalplugin.py", line 156, in __wait_server_to_start
    QMessageBox.Ok)
TypeError: 'Icon' object is not callable

6. Immediately after clicking ok on the Error pop up, it disappears and Sypder suddenly is responsive with no lag and works fine.

**What is the expected output? What do you see instead?**
For Spyder not to go unresponsive on startup, and not need to go through the steps listed above to make it work.

**Please provide any additional information below**
Again, the errors message lists the following:

  File "C:\Program Files\WinPython-64bit-3.5.4.0Qt5\python-3.5.4.amd64\lib\site-packages\spyder_terminal\terminalplugin.py", line 156, in __wait_server_to_start
    QMessageBox.Ok)
TypeError: 'Icon' object is not callable

## Version and main components

* Spyder Version: 3.2.1.dev0 
* Python Version: 3.5.4
* Qt Versions: 5.9.1, PyQt5 5.9 on Windows 7

## Dependencies
```
pyflakes >=0.6.0 :  1.6.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.20.3 (OK)
numpy >=1.7      :  1.13.1 (OK)
sphinx >=0.6.6   :  1.6.3 (OK)
rope >=0.9.4     :  0.10.5 (OK)
jedi >=0.9.0     :  0.10.2 (OK)
nbconvert >=4.0  :  5.2.1 (OK)
sympy >=0.7.3    :  1.1.1 (OK)
cython >=0.21    :  0.26 (OK)
qtconsole >=4.2.0:  4.3.0 (OK)
IPython >=4.0    :  6.1.0 (OK)
pylint >=0.25    :  1.7.2 (OK)

```

**Comment by rlaverde** [comment]
This is an spyder-terminal error already fixed in 0.2.1 https://github.com/spyder-ide/spyder-terminal/issues/96

Please wait until Winpython make a new release with a recent version of spyder-terminal

**Comment by jcbalmer** [comment]
Thanks, will do
